cmd/node/main.go

```diff
@@ -819,6 +828,11 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	err = ioutil.WriteFile(statsFile, []byte(sessionInfoFileOutput), os.ModePerm)
 	log.LogIfError(err)
 
+	computedRatingsData := filepath.Join(statsFolder, "ratings.info")
+	computedRatingsDataStr := createStringFromRatingsData(ratingsData)
+	err = ioutil.WriteFile(computedRatingsData, []byte(computedRatingsDataStr), os.ModePerm)
```

Comment:
is this really needed ?

Reply:
Currently it is for debugging purposes to see what was computed based on the shardSize, consensusSize, timeToReachMaxRating and proposerValidatorImportance. It is like the session.info.